src/documents/APIChangesFrom2_0To3_0.md

```diff
@@ -102,23 +99,205 @@ All panels must supply a `header` selector, which will be used by the `fluid.pre
     </thead>
     <tbody>
         <tr>
-            <td rowspan="2">speak.json</td>
+            <td rowspan="3">lineSpace.json</td>
             <td><code>"label"</code></td>
-            <td><code>"speakLabel"</code></td>
+            <td><code>"lineSpaceLabel"</code></td>
         </tr>
         <tr>
             <td><code>"description"</code></td>
-            <td><code>"speakDescr"</code></td>
+            <td><code>"lineSpaceDescr"</code></td>
+        </tr>
+        <tr>
+            <td></td>
+            <td><code>"multiplier"</code></td>
+        </tr>
+    </tbody>
+</table>
+
+##### Links and Buttons
+
+The "Links and Buttons" adjusters and enactors are collapsed to a single preference called "Enhance Inputs".
+
+###### Message Bundle Changes
+
+<table>
```

Comment:
Are message bundle changes and model changes mixed up in this table? The change from "textSize" to "value" looks like a model change while file names are message bundles.